features/api_token_info.feature

```diff
@@ -0,0 +1,151 @@
+Feature: api.u.pro.token_info.v1
+
+  @uses.config.contract_token
+  Scenario Outline: Get token info from api
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
```

Comment:
Let's also import this API in this test:
https://github.com/canonical/ubuntu-pro-client/blob/next-v35/features/api.feature#L3